document/core/util/macros.def

```diff
@@ -903,8 +899,7 @@
 .. |MITYPE| mathdef:: \xref{exec/runtime}{syntax-meminst}{\K{type}}
 .. |MIDATA| mathdef:: \xref{exec/runtime}{syntax-meminst}{\K{data}}
 
-.. |EVIATTR| mathdef:: \xref{exec/runtime}{syntax-eventinst}{\K{eventattr}}
-.. |EVITYPE| mathdef:: \xref{exec/runtime}{syntax-eventinst}{\K{type}}
+.. |EXNITYPE| mathdef:: \xref{exec/runtime}{syntax-exninst}{\K{type}}
```

Comment:
EITYPE is already used for an element instance (L. 907)

Reply:
Ah, right. Maybe we should turn that around, since exns are the more canonical thing now, and moreover, the type in an elem inst isn't even the inst's type. So perhaps rename that to EIELEMTYPE?

Reply:
Sounds good, it'll be in the next commit.

Reply:
Btw, I suppose do the same for ETYPE? so ETYPE-->EELEMTYPE and EXNTYPE-->ETYPE?

Reply:
Yes, makes sense.